fvm/environment/transaction_info.go

```diff
@@ -19,6 +20,25 @@ type TransactionInfoParams struct {
 	LimitAccountStorage    bool
 }
 
+type ScriptInfoParams struct {
+	ID     flow.Identifier
+	Script []byte
+}
+
+func NewScriptInfoParams(id flow.Identifier, code []byte) *ScriptInfoParams {
+	return &ScriptInfoParams{
+		ID:     id,
+		Script: code,
+	}
+}
+
+func NewScriptInfoParamsWithCode(code []byte) *ScriptInfoParams {
+	return &ScriptInfoParams{
+		ID:     flow.HashToID(hash.DefaultComputeHash(code)),
+		Script: code,
+	}
+}
```

Comment:
this file is named `transaction_info.go` maybe move this to its own file

Reply:
https://github.com/onflow/flow-go/pull/4648/commits/3459ec75b47550deb6d63955374dcad458324029